src/gameplay/game.ts

```diff
@@ -2283,8 +2300,73 @@ class Game extends Room {
     }
     return newRating;
   }
+
+  getPlayersOnMission(missionNum: number): Set<string> {      
+    const set = new Set<string>();
+    const VH = this.voteHistory; //brevity
+
+    if(VH === undefined) {
+      return set; // empty set.
+    }
+  
+    for (const player in VH) {
+      //TODO: write check to ensure player really is in this.PlayersInGame.
+      if (VH.hasOwnProperty(player)) {
+        if(VH[player].length < missionNum) { 
+          //in case mission hasn't happened yet.
+          return set;
+        }
+      
+        const missionNumVH = VH[player][missionNum - 1];
+        const lastPick = missionNumVH[missionNumVH.length - 1];
+  
+        if (lastPick.includes('VHpicked')) {
+          set.add(player);
+        }
+      }
+    }
+    return set;
+  }
+
+  updateMissionSizesSinad(): void {
+    // in 6p avalon, if m1 and m2 both succeed and m3 is a dani's pick (i.e. m3!=m2+1)
+    // then the sizes of m4 and m5 are swapped, requiring 4 ppl and 3 ppl respectively. 
+    console.log(this.getPlayersOnMission(2)) ;
+    console.log(this.getPlayersOnMission(3)) ;
+    console.log(!isSubsetOf(this.getPlayersOnMission(2),this.getPlayersOnMission(3)));
+    if(!isSubsetOf(
+         this.getPlayersOnMission(2)
+        ,this.getPlayersOnMission(3)
+      )) {
+        this.numPlayersOnMission[6 - MIN_PLAYERS] = [2,3,4,4,3];
```

Comment:
Can you run prettier to format the file pls? 

https://prettier.io/docs/en/cli.html

It should be installed when you run `yarn`, but if not, do `yarn prettier . --write`